.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 *.log*
 .idea
-cucumber.json
\ No newline at end of file
+cucumber.json
```

Comment:
If you execute the test in parallel way, the framework will create many cucumberXXXX.json files. They will be able to push to github. Could you please fix this problem?

Reply:
done